demos/classification_benchmark_demo/cpp_gapi/README.md

```diff
@@ -0,0 +1,183 @@
+# Classification Benchmark C++ G-API Demo
+
+![](../classification_benchmark.gif)
+
+The demo visualizes OpenVINO performance on inference of neural networks for image classification.
+
+## How It Works
+
+On startup, the application reads command line parameters and loads a classification model to OpenVINO™ Runtime plugin for execution. It might take some time for demo to read all input images. Then the demo performs inference to classify the images and places them on grid.
+
+The demo starts in "Testing mode" with fixed grid size. After calculating the average FPS result, it will switch to normal mode and grid will be readjusted depending on model performance. Bigger grid means higher performance. You can repeat testing by pressing "Space" or "R" button.
+
+When "ground truth" data is applied, the color coding for the text, drawn above each image, shows whether the classification was correct: green means correct class prediction, red means wrong.
+
+You can stop the demo by pressing "Esc" or "Q" button. After that, the average metrics values will be printed to the console.
+
+> **NOTE**: By default, Open Model Zoo demos expect input with BGR channels order. If you trained your model to work with RGB order, you need to manually rearrange the default channels order in the demo application or reconvert your model using the Model Optimizer tool with the `--reverse_input_channels` argument specified. For more information about the argument, refer to **When to Reverse Input Channels** section of [Embedding Preprocessing Computation](@ref openvino_docs_MO_DG_Additional_Optimization_Use_Cases).
+
+## Preparing to Run
+
+The list of models supported by the demo is in `<omz_dir>/demos/classification_benchmark_demo/cpp_gapi/models.lst` file.
+This file can be used as a parameter for [Model Downloader](../../../tools/model_tools/README.md) and Converter to download and, if necessary, convert models to OpenVINO IR format (\*.xml + \*.bin).
+
+An example of using the Model Downloader:
+
+```sh
+omz_downloader --list models.lst
+```
+
+An example of using the Model Converter:
+
+```sh
+omz_converter --list models.lst
+```
+
+### Supported Models
+
+* alexnet
```

Comment:
not tested. Do you have points that G-API cannot cover them automatically?
I mean - is there clear understanding that something wouldn't be supported?

Reply:
Well, classification models are pretty the same in terms of parsing output, just double check that your code working if there are `1001` class instead of `1000` just in case

Reply:
It was the reason why gapi benchmark gives different output initially. The fix if here https://github.com/openvinotoolkit/open_model_zoo/pull/3800/files#diff-e122ede2725aacb2bad4130ba9f3c7b9855157b24d61440bcd7a4985f3d2713dR29